aws/validators.go

```diff
@@ -1651,3 +1651,21 @@ func validateCognitoRoles(v map[string]interface{}, k string) (errors []error) {
 
 	return
 }
+
+func validateDxConnectionBandWidth(v interface{}, k string) (ws []string, errors []error) {
```

Comment:
How do you feel about renaming the other validation function from the previous PR and reusing it here? It seems to be validating the exact same thing.

Reply:
I think `validateDxConnectionBandWidth` is better name so I replace dx_lag validate func with it.